appinfo/signature.json

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
never add signature.json to git - please remove - THX

Reply:
Understood, I removed it.